rewriter/Minitest.cc

```diff
@@ -183,19 +276,28 @@ unique_ptr<ast::Expression> runSingle(core::MutableContext ctx, ast::Send *send)
         ConstantMover constantMover;
         send->block->body = ast::TreeMap::apply(ctx, constantMover, move(send->block->body));
         auto name = ctx.state.enterNameUTF8("<it '" + argString + "'>");
+        unique_ptr<ast::Expression> body = std::move(send->block->body);
+        // if we have context, then it means we've got a binding to add to the top of 'it'-blocks here: add it to the
+        // front of the body so we re-introduce that variable into scope
+        if (auto &c = context) {
+            ast::InsSeq::STATS_store ins;
+            ins.emplace_back((*c)->deepCopy());
```

Comment:
Non blocking preference to rewrite this with `.has_value()` and `.value()`, which makes it a little more clear to the reader that this is working with option types, rather than pointers to pointers.

Reply:
(Though I do wish that C++ had something more similar to Rust's `if let Some(c) = context { ... }` syntax, I find that the existing syntax obfuscates things more than it helps.)